Mathlib/NumberTheory/LSeries/HurwitzZetaValues.lean

```diff
@@ -197,3 +199,57 @@ theorem hurwitzZeta_neg_nat (hk : k ≠ 0) (hx : x ∈ Icc (0 : ℝ) 1) :
   · exact_mod_cast hurwitzZeta_one_sub_two_mul_nat (by omega : n + 1 ≠ 0) hx
 
 end HurwitzZeta
+
+open HurwitzZeta
+
+/-- Explicit formula for `ζ (2 * k)`, for `k ∈ ℕ` with `k ≠ 0`: we have
+`ζ (2 * k) = (-1) ^ (k + 1) * 2 ^ (2 * k - 1) * π ^ (2 * k) * bernoulli (2 * k) / (2 * k)!`.
+Compare `hasSum_zeta_nat` for a version formulated explicitly as a sum, and
+`riemannZeta_neg_nat_eq_bernoulli` for values at negative integers (equivalent to the above via
+the functional equation). -/
+theorem riemannZeta_two_mul_nat {k : ℕ} (hk : k ≠ 0) :
+    riemannZeta (2 * k) = (-1) ^ (k + 1) * (2 : ℂ) ^ (2 * k - 1)
+      * (π : ℂ) ^ (2 * k) * bernoulli (2 * k) / (2 * k)! := by
+  convert congr_arg ((↑) : ℝ → ℂ) (hasSum_zeta_nat hk).tsum_eq
+  · rw [← Nat.cast_two, ← Nat.cast_mul, zeta_nat_eq_tsum_of_gt_one]
+    · rw [ofReal_tsum]
+      norm_num
+    · refine one_lt_two.trans_le ?_
+      conv_lhs => rw [← mul_one 2]
+      rwa [mul_le_mul_left (zero_lt_two' ℕ), Nat.one_le_iff_ne_zero]
+  · norm_num
```

Comment:
```suggestion
  · rw [← Nat.cast_two, ← Nat.cast_mul, zeta_nat_eq_tsum_of_gt_one (by omega)]
    simp only [ofReal_tsum, ofReal_div, ofReal_pow, ofReal_natCast, ofReal_one]
  · norm_cast
```
`norm_num` looks out of scope (no numerical computation), and inequalities of natural numbers or integers can be (efficiently) dealt with using `omega`.

Reply:
Good point, I think I must have got `norm_num` and `norm_cast` muddled.